src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  showUser: Todo,
+  setShowUser: (todo: Todo | null) => void,
+  setSelectedTodoId: (id: number) => void,
+}
+
+export const TodoModal: React.FC<Props>
+= ({ showUser, setShowUser, setSelectedTodoId }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(showUser.userId).then((userToShow: User | null) => {
+      if (userToShow) {
+        setUser(userToShow);
+      }
+    });
+  });
```

Comment:
This use effect doesn't have deeps. Request be in all cases where be manipulation with page.